gemrb/core/Map.cpp

```diff
@@ -2607,7 +2629,7 @@ PathNode *Map::FindPath(const Point &s, const Point &d, unsigned int size, unsig
 	smptDest.x = d.x / 16;
 	smptDest.y = d.y / 12;
 
-	if (size && GetBlocked(d.x, d.y, size)) {
+	if (size && GetBlocked(d.x, d.y, size, true)) { // We don't want to bump a still npc out of position
```

Comment:
If I understand the code correctly, I think it would be clearer to put it as something like this:
```suggestion
	if (size && GetBlocked(d.x, d.y, size, true)) { // We don't want to bump, but to stop before, a still npc on our target position
```